src/instrumentation_measurement/measurements.py

```diff
@@ -185,17 +183,16 @@ def run_geth_benchmark(self, program, sample_size, exec_path):
         else:
             exec_path = os.path.abspath(exec_path)
 
-        args = ['--code', program.bytecode, '--bench', 'run']
+        args = ['run', '--bench', program.bytecode]
```

Comment:
Here's the important change . The other changes are a small refactor - there was an unused argument for `_parse_geth_benchmark` function.